### PR TITLE
add bella

### PR DESCRIPTION
**Pull request by alex-arpa**
Changes proposed in this pull request:
- Add Bella

### CODE REVIEW

File: spaces/bella/skin.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.name {
```
Comment (bonustrack) [inline spaces/bella/skin.scss:1]: This must be the id of your space
```suggestion
.bella {
```

File: spaces/bella/skin.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.bella {
+  --primary-color: #384aff;
```
Comment (bonustrack) [inline spaces/bella/skin.scss:2]: This skin seem exactly the same as the default skin. If that's the case we don't need this file in the PR, could you remove it?

Comment (alex-arpa) [reply spaces/bella/skin.scss]: removed the skin config

**Review by bonustrack — APPROVED** [review]
Looking good!

**Pull request merged**